ajax/libs/rrssb/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "rrssb",
+  "filename": "rrssb.min.js",
+  "version": "1.14.0",
+  "author": "Daniel Box <daniel@kurtnoble.com> (http://github.com/kni-labs)",
+  "description": "Ridiculously Responsive Social Sharing Buttons",
+  "keywords": [
+    "ridiculously",
+    "RRSSB",
+    "rrssb",
+    "responsive",
+    "sharing",
+    "social",
+    "buttons"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kni-labs/rrssb"
+  },
+  "license": "MIT",
+  "homepage": "https://rrssb.netlify.com/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kni-labs/rrssb.git",
+    "fileMap": [
+      {
+        "basePath": "js",
```

Comment:
We don't use `js` and `css` as the basePath.
Let's use `""` as the basePath.
Thanks.